unit-test-coverage/pc-rtems/src/coveragetest-cfe-psp-support.c

```diff
@@ -0,0 +1,63 @@
+/************************************************************************
+ * NASA Docket No. GSC-18,719-1, and identified as “core Flight System: Bootes”
+ *
+ * Copyright (c) 2020 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ************************************************************************/
+
+#include "cfe_psp.h"
+
+#include "coveragetest-psp-pc-rtems.h"
+
+#include "PCS_stdlib.h"
+#include "PCS_cfe_configdata.h"
+
+void Test_CFE_PSP_Restart(void)
+{
+    CFE_PSP_Restart(CFE_PSP_RST_TYPE_PROCESSOR);
+    UtAssert_STUB_COUNT(PCS_exit, 1);
+    UT_ResetState(UT_KEY(PCS_exit)); /* Reset so cleared for next test */
+
+    CFE_PSP_Restart(CFE_PSP_RST_TYPE_POWERON);
+    UtAssert_STUB_COUNT(PCS_exit, 1);
+    UT_ResetState(UT_KEY(PCS_exit)); /* Reset so cleared for future tests */
```

Comment:
`UT_ResetState` at end of test shouldn't be necessary, `Psp_Test_Setup` resets everything prior to each test.

Reply:
All the `ResetState's` you flagged have been removed.
From memory, the `malloc's` needed to be reset or some tests would fail if they were run in a certain order - like something was carrying over between tests, but it's working fine now without them so I guess if it becomes an issue it can be fixed in the future. Could also have been something else causing the issue that I was unaware of.
Anyway - all good for now ;-)